Remotion/Development/Web/ResourceHosting/FileExtensionHandlerMapping.cs

```diff
@@ -36,6 +36,7 @@ public static FileExtensionHandlerMapping[] Default
                    new FileExtensionHandlerMapping ("png", ResourceVirtualPathProvider.StaticFileHandler),
                    new FileExtensionHandlerMapping ("gif", ResourceVirtualPathProvider.StaticFileHandler),
                    new FileExtensionHandlerMapping ("js", ResourceVirtualPathProvider.StaticFileHandler),
+                   new FileExtensionHandlerMapping ("map", ResourceVirtualPathProvider.StaticFileHandler),
```

Comment:
that change should be in the last commit since we don't integrate the TS files before then.